cvmfs/statistics.h

```diff
@@ -218,7 +218,7 @@ class Recorder {
 
 
 /**
- * Writes to multiple recorders.  Recorders with coarsed-grained resolution and
+ * Writes to multiple recorders.  Recorders with coarse resolution and
```

Comment:
`coarse-grained` is a thing (~5 million hits in Google), but `coarse-grained resolution` is barely at a thing (at 10k hits).

?